processed 3 tasks

task 1 'run'. lines 14-14:
Error: Function execution failed with VMError: {
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: 0x2::a,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 3)],
}

task 2 'run'. lines 16-24:
Error: Script execution failed with VMError: {
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 3)],
}
